types/office-js/index.d.ts

```diff
@@ -93500,813 +94214,657 @@ declare namespace Visio {
     }
     /**
      *
-     * Represents the Shape class.
+     * Master Information
      *
      * [Api set:  1.1]
      */
-    class Shape extends OfficeExtension.ClientObject {
-        /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
-        context: RequestContext; 
-        /**
-         *
-         * Returns the Comments Collection. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly comments: Visio.CommentCollection;
-        /**
-         *
-         * Returns the Hyperlinks collection for a Shape object. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly hyperlinks: Visio.HyperlinkCollection;
-        /**
-         *
-         * Returns the Shape's Data Section. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly shapeDataItems: Visio.ShapeDataItemCollection;
-        /**
-         *
-         * Gets SubShape Collection. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly subShapes: Visio.ShapeCollection;
-        /**
-         *
-         * Returns the view of the shape. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly view: Visio.ShapeView;
+    interface MasterInfo {
         /**
          *
-         * Shape's identifier. Read-only.
+         * Localised Display Name of the Master
          *
          * [Api set:  1.1]
          */
-        readonly id: number;
+        name: string;
         /**
          *
-         * Shape's name. Read-only.
+         * Master Name
```

Comment:
Here and other Visio descriptions missing period at the end.
```suggestion
         * Master name.
```

Reply:
I did a quick pass to add periods and adjust capitalization. TBH, there are so many other issues (some existing, some new), that I think it's better to fix the base file and rerun codegen. I'm pushing this to get parity with the CDN, but prioritizing the actual update will be a different problem.